database/migrations/000060_repo_reminder.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+ALTER TABLE repositories ADD COLUMN reminder_last_sent TIMESTAMP;
+
+CREATE EXTENSION IF NOT EXISTS tsm_system_rows;
```

Comment:
I'm not sure if enabling extensions should be part of migration files.

Reply:
suggestion: I think we can get rid of query `GetRandomRepository` by calculating a random UUID on the application side and passing it straight to `ListEligibleRepositoriesAfterID`. Added benefits would be
* we have one less query
* one less step in the process
* the whole process becomes deterministic (and easier to test)

Reply:
If we want to start at a random point, fetching a valid random repo is required. If we can start at any point, then sequential iteration would do the job.

If we don't fetch a valid repo / generate a random uuid, then we can potentially have a case where the generated uuid is out of range, and we have to either generate a uuid again or start from the beginning (which defeats the purpose of random generation)

It all comes down to whether we want to start from a valid random point or not.